nifi-nar-bundles/nifi-gcp-bundle/nifi-gcp-processors/src/main/java/org/apache/nifi/processors/gcp/storage/DeleteGCSObject.java

```diff
@@ -81,6 +81,7 @@ public class DeleteGCSObject extends AbstractGCSProcessor {
     public List<PropertyDescriptor> getSupportedPropertyDescriptors() {
         final List<PropertyDescriptor> descriptors = new ArrayList<>();
         descriptors.addAll(super.getSupportedPropertyDescriptors());
+        descriptors.add(STORAGE_API_HOST);
```

Comment:
Instead of adding the Property Descriptor in individual components, it looks like it could be added after the Proxy Configuration Service in the AbstractGCSProcessor.